CONTRIBUTING.md

```diff
@@ -76,3 +76,113 @@ issues.
+ └─ build/                          # *nix build system files
+    ├─ ax_check_compile_flag.m4     # https://github.com/autoconf-archive/autoconf-archive
+    ├─ ax_gcc_func_attribute.m4     # https://github.com/autoconf-archive/autoconf-archive
+    ├─ libtool.m4                   # https://git.savannah.gnu.org/cgit/libtool.git
+    ├─ shtool                       # https://www.gnu.org/software/shtool/
+    └─ ...
+ └─ ext/                            # PHP core extensions
+    └─ bcmath/
+       ├─ libbcmath/                # Forked and maintained in php-src
+       └─ ...
+    └─ date/
+       └─ lib/                      # Bundled datetime library https://github.com/derickr/timelib
+          ├─ parse_date.c           # Generated by re2c 0.15.3
+          ├─ parse_iso_intervals.c  # Generated by re2c 0.15.3
+          └─ ...
+       └─ ...
+    └─ fileinfo/
+       ├─ libmagic/                 # Bundled libmagic https://github.com/file/file/
+       ├─ data_file.c               # Generated by `ext/fileinfo/create_data_file.php`
+       └─ ...
+    └─ gd/
+       ├─ libgd/                    # https://github.com/libgd/libgd
+       └─ ...
+    └─ json/
+       ├─ json_scanner.c            # Generated by re2c 1.0.3
+       ├─ php_json_scanner_defs.h   # Generated by re2c 1.0.3
+       ├─ json_parser.tab.c         # Generated by bison
+       ├─ json_parser.tab.h         # Generated by bison
+       └─ ...
+    └─ mbstring/
+       ├─ libmbfl/                  # Forked and maintained in php-src
+       ├─ onigoruma/                # https://github.com/kkos/oniguruma
+       ├─ unicode_data.h            # Generated by `ext/mbstring/ucgendat/ucgendat.php`
+       └─ ...
+    └─ pcre/
+       ├─ pcre2lib/                 # https://www.pcre.org/
+       └─ ...
+    └─ pdo/
+       ├─ pdo_sql_parser.c          # Generated by re2c 1.0.3
+       └─ ...
+    └─ phar/
+       ├─ phar_path_check.c         # Generated by re2c 1.0.3
+       └─ ...
+    └─ sqlite3/
+       ├─ libsqlite/                # https://www.sqlite.org/src/dir Mirror: https://github.com/mackyle/sqlite
+       └─ ...
+    └─ standard/
+       └─ html_tables/
+          ├─ mappings/              # https://www.unicode.org/Public/MAPPINGS/
+          └─ ...
+       ├─ url_scanner_ex.c          # Generated by re2c 1.0.3
+       ├─ var_unserializer.c        # Generated by re2c 1.0.3
+       ├─ credits_ext.h             # Generated by `scripts/dev/credits`
+       ├─ credits_sapi.h            # Generated by `scripts/dev/credits`
+       ├─ html_tables.h             # Generated by `ext/standard/html_tables/html_table_gen.php`
+       └─ ...
+    └─ xmlrpc/
+       ├─ libxmlrpc/                # Forked and maintained in php-src
+       └─ ...
+    └─ zip/                         # Bundled https://github.com/pierrejoye/php_zip
+       ├─ lib/                      # Bundled https://libzip.org/
+       └─ ...
+    └─ ...
+ ├─ main/                           # Language core features
```

Comment:
I'm not sure about this one, but also can't figure out a better explanation. Core’s feature is everywhere, not only here :(

Reply:
I've changed this to `Main macros and definitions` as noted in the [manual](https://php.net/manual/internals2.ze1.zendapi.php) Thanks for reviewing.

Reply:
@carusogabriel @petk It could probably be noted better that its the binding that ties extensions, SAPIs and the Engine together (and implements the stream layer subsystem), just worded way better than what I just did ^_^

Reply:
Thanks for this info. Sure, directory structures are a usual part of code projects READMEs and such docs files also. I've extended the main directory into the following then which might explain it ok:

```bash
 └─ main/                           # Binding that ties extensions, SAPIs, and engine together
    ├─ streams/                     # Stream layer subsystem
    └─ ...
```